daprdocs/content/en/operations/configuration/configuration-overview.md

```diff
@@ -108,19 +108,34 @@ The `metrics` section under the `Configuration` spec contains the following prop
+| Property                     | Type    | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+|------------------------------|---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `enabled`                    | boolean | When set to true, the default, enables metrics collection and the metrics endpoint.                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `rules`                      | array   | Named rule to filter metrics. Each rule contains a set of `labels` to filter on and a `regex` expression to apply to the metrics path.                                                                                                                                                                                                                                                                                                                                                                                                           |
+| `latencyDistributionBuckets` | array   | Array of latency distribution buckets in milliseconds for latency metrics histograms.                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
```suggestion
| `latencyDistributionBuckets` | array   | Array of latency distribution buckets in milliseconds for latency metrics histograms.  A latency distribution bucket is a sampling rate for the metric, so that you can observe a metrics over a given time period for example over the last 1 second.                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Reply:
Please check my first pass description of a latency bucket.

Reply:
@msfussell could you take another look to a revised verion?